types/react/v17/index.d.ts

```diff
@@ -26,6 +26,7 @@
 //                 Victor Magalhães <https://github.com/vhfmag>
 //                 Dale Tan <https://github.com/hellatan>
 //                 Priyanshu Rav <https://github.com/priyanshurav>
+//                 Costa Alexoglou <https://github.com/konsalex>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
```

Comment:
Should it be updated to `// TypeScript Version: 3.5`?

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-5.html#the-omit-helper-type

Reply:
@Semigradsky thought the same, but not sure how it worked, as I can see they left it unchanged in the v18 types: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L31

Should I change?

Reply:
The comment is only meaningful for versions greater than 4.4: at least when TS 5.1 is out this week, DT will update to only support TS 4.4 and above.

If anything, the comment should be deleted. (There's no lint rule for this because it would make editing old packages too annoying.)